website/oauth/models/__init__.py

```diff
@@ -170,6 +170,9 @@ def auth_url(self):
             oauth = OAuth1Session(
                 client_key=self.client_id,
                 client_secret=self.client_secret,
+                callback_uri=web_url_for('oauth_callback',
```

Comment:
Can you verify 

1. that Evernote doesn't work without this (which seems probable, given how OAuth is supposed to work) 
2. that this doesn't break Zotero, the other OAuth1 provider?